libs/wingc/src/lsp/completions.rs

```diff
@@ -178,8 +178,10 @@ pub fn on_completion(params: lsp_types::CompletionParams) -> CompletionResponse
 			completions.push(format_symbol_kind_as_completion(symbol_data.0, symbol_kind));
 		}
 
-		for data in found_env.iter(true) {
-			completions.push(format_symbol_kind_as_completion(&data.0, &data.1));
+		if let Some(parent) = found_env.parent {
```

Comment:
where we adding symbols from the current env twice up until now?
I suspect we need to use the `found_env.statemet_idx` and compare it against our parent's satement indexes and follow this all the way up the env tree. We might need to extend env's iter to support this.

Reply:
Yep, it was an existing bug.

> I suspect we need to use the found_env.statemet_idx and compare it against our parent's satement indexes and follow this all the way up the env tree. We might need to extend env's iter to support this.

Yeah I agree, although I think it would be okay to keep that for another issue/PR. I only fixed the other bug since it was a big low-hanging fruit I ran into.

Reply:
can you verify this really happens and open and issue?

Reply:
https://github.com/winglang/wing/issues/1984